apps/condo/domains/acquiring/hooks/usePaymentsTableFilters.ts

```diff
@@ -113,6 +115,17 @@ export function usePaymentsTableFilters (
                     },
                 },
             },
+            {
+                keyword: 'order',
+                filters: [orderFilter],
+                component: {
+                    type: ComponentType.Input,
+                    modalFilterComponentWrapper: {
+                        label: paymentOrderTitle,
```

Comment:
How about fix useMemo deps?)
Seems like it missing new translation as dep (and other stuff as well)